jwt.d.ts

```diff
@@ -48,7 +48,7 @@ export interface VerifyCallback<Decoded extends VerifyPayloadType> extends jwt.V
 }
 
 export interface FastifyJWTOptions {
-  secret: Secret | { public: Secret; private: Secret }
+  secret: Secret | { public: Secret; private: Secret } | Promise<void>

```

Comment:
the promise should resolve to a Secret or more specifically to a jwt.Secret

Reply:
Changed to jwt.Secret